### PR TITLE
Remove references to a specific SQL version to make the guidance vers…

### PR DESCRIPTION
**Pull request by andreiztm**
…ion agnostic

This way, we do not have to revise/edit the article when SQL Express 2017 goes out of support.
Correcting some typos.

### CODE REVIEW

File: windows/deployment/volume-activation/install-vamt.md [hunk 1]
```diff
@@ -34,12 +34,12 @@ You install VAMT as part of the Windows Assessment and Deployment Kit (ADK) for
 
 - [Windows Server with Desktop Experience](https://docs.microsoft.com/windows-server/get-started/getting-started-with-server-with-desktop-experience), with internet access (for the main VAMT console) and all updates applied
 - [Windows 10, version 1903 ADK](https://go.microsoft.com/fwlink/?linkid=2086042)
-- [SQL Server 2017 Express](https://www.microsoft.com/sql-server/sql-server-editions-express)
-- alternatively any full SQL instance e.g. SQL Server 2014 or newer incl. CU / SP
+- any supported [SQL Server Express](https://www.microsoft.com/sql-server/sql-server-editions-express) version, the latest is recommneded
+- alternatively any full SQL instance that is supported
```
Comment (illfated) [inline windows/deployment/volume-activation/install-vamt.md:38]: suggestion for a readability improvement / more easily spoken form:
```suggestion
- alternatively any supported **full** SQL instance
```

**Comment by andreiztm** [comment]
Thanks for the review @illfated I think I merged your suggestions correctly.

Thanks,
Andrei

**Comment by illfated** [comment]
Looks quite good. Only one minor detail, which only shows up in the rich diff and the edited file view:

5. On the completion page, note the instance name for your installation, select **Close**, and then select **Yes**. 
    In this example, the instance name is SQLEXPRESS01


```diff
  5. On the completion page, note the instance name for your installation, select **Close**, and then select **Yes**. 
+                                                                                                                                                                                               
    ![In this example, the instance name is SQLEXPRESS01](images/sql-instance.png)
```

Would you be able to insert a necessary blank line in between line number 5 and the image, to have the image displayed below the text, instead of at the end of the text line?
(I don't have suggestion access to those, they are _just_ outside of the edited area.)

I know, it looks OK here in this comment, but see for yourself how it looks here:
https://github.com/MicrosoftDocs/windows-itpro-docs/blob/public/windows/deployment/volume-activation/install-vamt.md

**Comment by illfated** [comment]
As a matter of fact, I can make a PR to your branch [`andreiztm:patch-33`](https://github.com/andreiztm/windows-itpro-docs/tree/patch-33) if you find it troublesome to make the edit manually.

The file to edit in this case is here: https://github.com/andreiztm/windows-itpro-docs/blob/patch-33/windows/deployment/volume-activation/install-vamt.md

**Comment by andreiztm** [comment]
@illfated thanks for the suggestion, didn't notice that. I changed it now, how does it look?

**Comment by illfated** [comment]
#sign-off

**Comment by andreiztm** [comment]
Thank you for the help! :D

**Comment by Karl-WE** [comment]
Since this has gone live I will revoke PR #6066 and add some of the information in a new PR

**Pull request merged**